gensim/corpora/sharded_corpus.py

```diff
@@ -647,15 +647,15 @@ def __getitem__(self, offset):
             #    to (stop - current_offset)
             shard_start = start - self.current_offset
             shard_stop = self.offsets[self.current_shard_n + 1] - \
-                         self.current_offset
+                self.current_offset
```

Comment:
@piskvorky 
These lines were already split. Here `autopep8` corrected the vertical indentation.
Though we can write a script to join these kind of lines, but that would result into many counter-intuitive fixings where lines are split for documentation purposes.